robot/Cumulus/resources/LeadsPageObject.py

```diff
@@ -0,0 +1,45 @@
+from cumulusci.robotframework.pageobjects import ListingPage
+from cumulusci.robotframework.pageobjects import DetailPage
+from cumulusci.robotframework.pageobjects import pageobject
+from cumulusci.robotframework.pageobjects import BasePage
+from BaseObjects import BaseNPSPPage
+from NPSP import npsp_lex_locators
+
+
+@pageobject("Custom", "Lead")
+class CovertLeadPage(BaseNPSPPage, BasePage):
+
+    def _is_current_page(self):
+        """
+        Waits for the Leads iframe to load
+        """
+        locator = '//iframe[contains(@name,"vfFrameId")]'
+        self.selenium.wait_until_element_is_visible(locator, timeout=60)
+        frame = self.selenium.driver.find_element_by_xpath(locator)
+        self.selenium.driver.switch_to.frame(frame);
```

Comment:
there is already an existing locator for same in locator file. Also cant we use select frame or choose frame keywords here?

Reply:
Here the iframe name is changed to something like "vfFrameId_123455" . so instead of hardcoding, I am using iframe name contains. I was not able to find the exact locator in the locators file. I also tried to choose frame keyword and that did not work so this alternative solution.

Reply:
u can use self.npsp.choose_frame("vfFrameId") that was the base keyword for selecting frame inside the original keyword in testcase.

Reply:
Tried and some reason did not work

Reply:
updated